src/subapps/admin/components/Settings/ViewsSubView.tsx

```diff
@@ -392,42 +435,72 @@ const ViewsSubView = () => {
           </Col>
         </Row>
 
-        <Table<TViewType>
+        <Table<SubView>
           loading={status === 'loading'}
           className="views-table"
           rowClassName="view-item-row"
           columns={columns}
-          dataSource={views?.results}
+          dataSource={views?.results as SubView[]}
           sticky={true}
           size="middle"
           pagination={false}
           rowKey={r => r.key}
           expandIcon={({ expanded, onExpand, record }) =>
             expanded ? (
-              <MinusCircleTwoTone
-                title="Collapse indexing errors"
-                onClick={e => onExpand(record, e)}
-              />
+              <MinusCircleTwoTone onClick={e => onExpand(record, e)} />
             ) : (
-              <Badge
-                count={record.indexingErrors._total}
-                showZero={false}
-                size="small"
-              >
-                <PlusCircleTwoTone
-                  title="Expand indexing errors"
-                  data-testid="Expand indexing errors"
-                  onClick={e => onExpand(record, e)}
-                  style={{ fontSize: '16px' }}
-                />
-              </Badge>
+              <PlusCircleTwoTone
+                data-testid="Expand indexing errors"
+                onClick={async e => {
+                  // Fetch indexing errors when expanding the row
+                  // We do this on demand to avoid fetching all indexing errors for all views at once (which can be a lot)
+                  record.indexingErrors = await fetchIndexingErrorsOnDemand({
+                    nexus,
+                    apiEndpoint,
+                    orgLabel: record.orgLabel,
+                    projectLabel: record.projectLabel,
+                    viewId: record.id,
+                  });
+                  onExpand(record, e);
+                }}
+              />
             )
           }
-          expandedRowRender={(r: TViewType) => {
+          expandedRowRender={(r: SubView) => {
+            if (!expandedRows[r.key]) {
+              // Fetch errors if not already fetched for this row
+              fetchIndexingErrorsOnDemand({
+                nexus,
+                apiEndpoint,
+                orgLabel: r.orgLabel,
+                projectLabel: r.projectLabel,
+                viewId: r.id,
+              })
+                .then(indexingErrors => {
+                  // Update the state with the new errors
+                  setExpandedRows(prevExpandedRows => ({
+                    ...prevExpandedRows,
+                    [r.key]: indexingErrors as IndexingErrorResults,
+                  }));
+                })
+                .catch(error => {
+                  console.error(
```

Comment:
Same question here about why we need this console (I'm a bit picky with these console errors because they don't really help us debug issues in production since the error is only logged on user's browser's console that the developer doesn't have access to, and they add too much noise in the console when we are developing. If for some reason, the error tracing is required, we should instead use Sentry maybe?)

Reply:
I noticed console errors/logs scattered throughout. I believe this is legacy code or noise. In my experience, we always added a linter rule to display a warning when there is a console.log statement, as it provides no benefit. Therefore, I suggest removing it.